src/breadcrumb/breadcrumb.stories.ts

```diff
@@ -0,0 +1,87 @@
+import { storiesOf, moduleMetadata } from "@storybook/angular";
+import { withNotes } from "@storybook/addon-notes";
+import { action } from "@storybook/addon-actions";
+import { withKnobs, boolean, number } from "@storybook/addon-knobs/angular";
+
+import { BreadcrumbModule, DialogModule } from "../";
+
+storiesOf("Breadcrumb", module)
+	.addDecorator(
+		moduleMetadata({
+			imports: [BreadcrumbModule, DialogModule]
+		})
+	)
+	.addDecorator(withKnobs)
+	.add("Basic", () => ({
+		template: `
+		<ibm-breadcrumb [noTrailingSlash]="noTrailingSlash">
+			<ibm-breadcrumb-item href="#">
+				Breadcrumb 1
+			</ibm-breadcrumb-item>
+			<ibm-breadcrumb-item href="#">
+				Breadcrumb 2
+			</ibm-breadcrumb-item>
+			<ibm-breadcrumb-item>
+				Breadcrumb 3
+			</ibm-breadcrumb-item>
+		</ibm-breadcrumb>`,
+		props: {
+			noTrailingSlash: boolean("noTrailingSlash", true)
+		}
+	}))
+	.add("Static Overflow", () => ({
```

Comment:
We shouldn't encourage this usecase and this story should be removed now that the full solution is available.